chapters/1.md

```diff
@@ -1,100 +1,185 @@
-# Chapter 1 
-## Introduction
+# Chapter 1
 
-## An Introduction to Cryptocurrency and Privacy
+# Introduction
 
-Meet Leo and George. George would like to go on vacation, but he has not saved enough and has limited access to money. Leo, who is George’s best friend, is kind enough to lend George $2,500 to pay for a flight, hotel room, and food.
+## How can we send money to a friend?
 
-In traditional banking, in order for Leo to send George the $2,500, Leo would ask his bank to send the money to George’s bank. Leo would tell George that the transaction has taken place, and George would then be able to withdraw the needed $2,500 for his vacation. In this transaction, Leo and George have trusted their banks to manage the money.
+Meet Maria and her best friend George, who decide to meet up for a vacation. George’s budget is tight,
+so Maria offers to send some money to help pay for the flight, hotel room, and food.
 
-There was no real movement of physical bills that took place during the transaction. Digitally speaking, the banks edited a centralized database that now shows $2,500 being sent from Leo to George. Therein lies the problem: to establish trust between Leo and George, they had to depend on a third party: the banks. This problem is also present in the multitude of transactions happening every day, as many of those transactions cannot occur without some third party.
+If Maria wires the money to George through the traditional banking system, they trust two intermediate parties (their respective banks) to symbolically move the funds for them. 
 
-This poses a risk, because a bank (or a nefarious actor) can create an infinite amount of money by editing the transaction database or altering your own bank account - along with other illicit acts.
+There is no actual movement of physical bills or assets, simply edits to the banks’ databases. When Maria announces the transaction “Maria is sending $2,500 to George”, her bank subtracts $2,500 from her account balance on their ledger, then contacts George’s bank and requests that they add $2,500 to George’s balance, so that he can withdraw the funds to pay for travel expenses. 
```

Comment:
s/`simply edits to the banks’ databases.`/`both banks simply edit their respective database to show that the funds have been moved.`

`When Maria announces the transaction “Maria is sending $2,500 to George”` 
This is an awkward way to state this. 
Possibly, `When Maria submits the transaction to her bank (whether that's online, on her bank app, or a check),`